src/components/RadioButton/RadioButton.ts

```diff
@@ -9,30 +9,59 @@ namespace fabric {
    * Adds basic demonstration functionality to .ms-RadioButton components.
    *
    */
-  export class RadioButton extends CheckBox {
+  export class RadioButton {
```

Comment:
Phil can provide more detail, but I think the issue he was seeing was that there's a delay when JS turns a checkbox into a radio button. So it appears as a checkbox for a second and then turns into a radio button.